resources/public/api/conf/2.0/application.yaml

```diff
@@ -0,0 +1,10973 @@
+            "transferredFromAccountId": "8765432100",
+            "transferredFromLMRN": "Z654321",
+            "transferInDate": "2017-04-06"
+          }
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "PREVIOUS_INVESTOR_ACCOUNT_DOES_NOT_EXIST",
+          "message": "The transferredFromAccountId and transferredFromLMRN given do not match an account on HMRC’s records"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Transfer request without transfer details</p><p class ="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "investorId": "9876543210",
+          "creationReason": "Transferred",
+          "accountId": "1234567890",
+          "firstSubscriptionDate": "2017-04-06"
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "TRANSFER_ACCOUNT_DATA_NOT_PROVIDED",
+          "message": "You must give a transferredFromAccountId, transferredFromLMRN and transferInDate when the creationReason is transferred, current or previous year funds transferred"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Create request containing transfer details</p><p class ="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
+                    <td>
+        <pre class="code--block">
+        {
+          "investorId": "9876543210",
+          "creationReason": "New",
+          "accountId": "1234567890",
+          "firstSubscriptionDate": "2017-04-06",
+          "transferAccount": {
+            "transferredFromAccountId": "8765432100",
+            "transferredFromLMRN": "Z654321",
+            "transferInDate": "2017-04-06"
+          }
+        }
+        </pre>
+                    </td>
+                    <td><p>HTTP status: <code class="code--slim">403 (Forbidden)</code></p>
+        <pre class="code--block">
+        {
+          "code": "TRANSFER_ACCOUNT_DATA_PROVIDED",
+          "message": "You must only give a transferredFromAccountId, transferredFromLMRN, and transferInDate when the creationReason is transferred, current or previous year funds transferred"
+        }
+        </pre>
+                    </td>
+                </tr>
+                <tr>
+                    <td><p>Request containing a LISA account which has already been closed</p><p class ="code--block">lisaManagerReferenceNumber: <a href="#section/Testing">Use your test user profile</a></p></td>
```

Comment:
Once the duplicated Testing section in this file is removed, please update link from`#section/Testing` to `/api-documentation/docs/api/service/lisa-api/2.0#testing`. Thanks